utils/db_tracker.py

```
@@ -43,53 +48,67 @@ def ensure_tables_exist():
+            CREATE INDEX IF NOT EXISTS idx_executions_time
             ON protocol_executions(execution_time);
-        """)
-        
+        """
+        )
+
         conn.commit()
         log("Database tables initialized successfully")
-        
+
     except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)